libs/application/templates/health-insurance/README.md

```diff
@@ -0,0 +1,9 @@
+<!-- gitbook-ignore -->
+
+# Application for health insurance
+
+Application for health insurance when moving back to Iceland after living abroad.
```

Comment:
i am wondering, does a general application for health insurance exist somewhere for Sjúkratryggingar?

My concern is that maybe this application you are working on is specific to health insurance for people moving back to iceland, so the HEALTH_INSURANCE enum value is "taken".

Reply:
I have investigated it and it seems that this is the general application. If an Icelandic citizen never moves abroad they will get health insurance automatically and doesn't need to apply for it. But I could always make this enum specific anyway if we want to be safe. What do you think?

For more background, there will be applications in the future for insurance claim from the Health insurance for example sick pay and work accidents. But that might not be conflicting with this enum as they are more specific.

Reply:
Good comment ^

Reply:
great answer, I think that this enum value (HEALTH_INSURANCE) can safely be reserved for this application.